src/Akeneo/Bundle/ElasticsearchBundle/Command/ResetIndexesCommand.php

```diff
@@ -34,6 +35,7 @@ protected function configure()
                 InputOption::VALUE_NONE,
                 'Resets all registered ES indexes prior to reindex'
             )
+            ->addOption('index', 'i', InputOption::VALUE_REQUIRED | InputOption::VALUE_IS_ARRAY, 'ES index name to reset')
             ->setDescription('Resets all registered ES indexes');
```

Comment:
```suggestion
            ->setDescription('Resets registered ES indexes');
```
Removing 'all' because it's not always the case.